examples/flutter_gallery/lib/demo/playground/home.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+
+import 'playground_demo.dart';
+
+class PlaygroundScaffold extends StatelessWidget {
+  const PlaygroundScaffold({
+    this.title,
+    this.demos,
+  });
+
+  final String title;
+  final List<PlaygroundDemo> demos;
+
+  static final GlobalKey<ScaffoldState> _scaffoldKey =
+      GlobalKey<ScaffoldState>();
+
+  @override
+  Widget build(BuildContext context) {
+    return DefaultTabController(
+      length: demos.length,
+      child: Scaffold(
+        key: _scaffoldKey,
+        appBar: AppBar(
+          title: Text(title),
+          leading: IconButton(
```

Comment:
Do you need to do this? Doesn't the app bar do this automatically?

Reply:
The `IconButton` does not seem to show automatically in this case. I was surprised as that was the expected behavior. There are existing areas of the gallery codebase that have similar code though. Open to suggestions.